analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0.spdx.yaml

```diff
@@ -0,0 +1,2125 @@
+    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+    OF THE POSSIBILITY OF SUCH DAMAGE. '
+  licenseId: LicenseRef-BSD-3-Clause
+  name: BSD 3-Clause "New" or "Revised" License
+- extractedText: ' License by Scancode. '
+  licenseId: LicenseRef-bsd-new
+  name: BSD-3-Clause
+- extractedText: ' BSD is referenced without a version number. Please look up BSD
+    in the License Admin to view the different versions. '
+  licenseId: LicenseRef-BSD
+  name: BSD
+- extractedText: ' Software License
+
+    Copyright (c) 2010, Google Inc. All rights reserved.
+
+
+    Redistribution and use in source and binary forms, with or without modification,
+    are permitted provided that the following conditions are met:
+
+
+    Redistributions of source code must retain the above copyright notice, this list
+    of conditions and the following disclaimer.
+
+
+    Redistributions in binary form must reproduce the above copyright notice, this
+    list of conditions and the following disclaimer in the documentation and/or other
+    materials provided with the distribution.
+
+
+    Neither the name of Google nor the names of its contributors may be used to endorse
+    or promote products derived from this software without specific prior written
+    permission.
+
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+    ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+    IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+    INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+    DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+    OF THE POSSIBILITY OF SUCH DAMAGE. '
+  licenseId: LicenseRef-WebM
+  name: WebM Software License
+name: /srv/fossology/repository/report
+packages:
+- SPDXID: SPDXRef-upload120
+  checksums:
+  - algorithm: SHA1
+    checksumValue: 7d861806e2669c0a67e16738070d05828bb849eb
+  - algorithm: SHA256
+    checksumValue: 212c3e62fcef8cb976c62848b193c82f2c74f0cfa398a6838dfc8ca4fa96ee5c
+  - algorithm: MD5
+    checksumValue: ac11fb6ae2292145e88eda4908f7ade0
+  copyrightText: NOASSERTION
+  downloadLocation: NOASSERTION
+  filesAnalyzed: true
+  licenseComments: " licenseInfoInFile determined by Scanners:\n - nomos (\"4.4.0\"\
+    .903df0)\n - monk (\"4.4.0\".903df0)\n - ojo (\"4.4.0\".903df0)\n - scancode (\"\
+    4.4.0\".903df0) "
+  licenseConcluded: NOASSERTION
+  licenseDeclared: LicenseRef-BSD-3-Clause
```

Comment:
Please correct the PacvkageLicenseDeclared, it must be:
LicenseRef-BSD-3-Clause-5d180954454293e3705aecb03706c1e6
Instead of LicenseRef-BSD-3-Clause